api/src/services/__tests__/programsService.ts

```diff
@@ -1,10 +1,11 @@
 import {
-  listPrograms,
+  listAllPrograms,
```

Comment:
Good job changing the variable name for better understanding

Reply:
Thank you so much! That was incorporated based on feedback from Davey, and I wholeheartedly agree.